FESS-GUI/mainwindow.cpp

```diff
@@ -153,7 +157,6 @@ void MainWindow::realtimeDataSlot()  //Important function. This is repeatedly ca
 
     if (currentTime-lastPointTime > 0.01) // at most add point every 10 ms
```

Comment:
This should be unnecessary, right? Since the dataTimer only calls this function once every 10 ms